source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/digit/loco_manip_env_cfg.py

```diff
@@ -0,0 +1,235 @@
+# Copyright (c) 2022-2025, The Isaac Lab Project Developers.
```

Comment:
To me this task doesn't seem to fit well under "velocity" set of tasks. I suggest moving it outside to `locomanipulation` folder at the same level as `locomotion` and we call the task something like `tracking`. Locomotion velocity is only for walking :)